jobs/invitations/RemoveExpiredInvitationsJob.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * @file jobs/invitations/RemoveExpiredInvitationsJob.php
+ *
+ * Copyright (c) 2014-2022 Simon Fraser University
```

Comment:
All new code should be `Copyright (c) 2023`.

Reply:
I changed the copyrights to 

```
Copyright (c) 2023 Simon Fraser University
Copyright (c) 2023 John Willinsky
```

for all added/new files introduced in this PRs